public/javascripts/ngApp.js

```diff
@@ -38,6 +38,7 @@ springInitiative.config(function($stateProvider, $urlRouterProvider,
       }
     }
   })
+  // Note: this state won't be viewed by itself anymore
```

Comment:
it's a parent of the others and i think the controller for it has important code, so we probably should